plugins/inventory/nb_inventory.py

```diff
@@ -450,6 +451,9 @@ def extract_disk(self, host):
     def extract_vcpus(self, host):
         return host.get("vcpus")
 
+    def extract_status(self, host):
+        return host.get("status")
```

Comment:
Should we return either the label or value from the `status` dictionary?

Reply:
Why limit ourselves? In doubt I'd prefer "value".

Reply:
This can be resolved by @metabsd's comment on the PR.